client/headers/casper_client.h

```diff
@@ -0,0 +1,31 @@
+#include <stdarg.h>
```

Comment:
So this file is generated based on the `pub const`s and `pub extern "C"`s?

Reply:
Yeah, I didn't want to do this by hand. It's my first time using it, but `cbindgen` looks pretty decent. https://github.com/eqrion/cbindgen

Reply:
I've added an option to build.rs' invocation of `cbindgen` to add the message `/* WARNING: this file is autogenerated by cbindgen. Don't modify this manually. */` to the generated file, as well as explicitly `#include` headers, along with an include guard.